tb_plugin/torch_tb_profiler/run.py

```diff
@@ -124,19 +125,50 @@ def append_gpu_metrics(self, raw_data):
         return raw_data
 
 
+    def get_gpu_info(self, gpu_id):
```

Comment:
could we reuse the _generate_gpu_info in run_generator.py?

Reply:
since most of the code same with that function

Reply:
Reasonable. Should reuse. Done. Thanks!